### PR TITLE
[bridgeagg] Modify list unified token and convert token to unified token

### PR DESCRIPTION
**Pull request by tin-incognito**
Add:
- Rpc modify list token
- Rpc convert token to unified token
- metadata modify list token
- metadata convert to unified token
- Data structure for vault manager

### CODE REVIEW

File: blockchain/bridgeagg/state.go [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+	}
+}
+
+func (s *State) Clone() *State {
+	res := NewState()
+	res.processor = stateProcessor{}
+	res.producer = stateProducer{}
+	for unifiedTokenID, vaults := range s.unifiedTokenInfos {
+		res.unifiedTokenInfos[unifiedTokenID] = make(map[uint]*Vault)
+		for tokenID, vault := range vaults {
```
Comment (0xkraken) [inline blockchain/bridgeagg/state.go:43]: It means networkID?

File: blockchain/bridgeagg/state.go [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+			res = append(res, inst)
+		}
+	}
+
+	Logger.log.Info("bridgeagg instructions:", res)
+
+	return res, nil
+}
+
+func (s *State) Process(insts [][]string, sDB *statedb.StateDB) error {
```
Comment (0xkraken) [inline blockchain/bridgeagg/state.go:105]: Should move reading config unified token above processing intructions?

File: rpcserver/http_bridge_agg.go [hunk 3]
```diff
@@ -0,0 +1,534 @@
...
+		return nil, rpcservice.NewRPCError(rpcservice.UnexpectedError, err1)
+	}
+	return res, nil
+}
+
+func (httpServer *HttpServer) createBridgeAggModifyListTokensTransaction(
+	params interface{},
+) (interface{}, *rpcservice.RPCError) {
+	arrayParams := common.InterfaceSlice(params)
+	privateKey, ok := arrayParams[0].(string)
```
Comment (0xkraken) [inline rpcserver/http_bridge_agg.go:41]: Check len(arrayParams) before accessing to avoid out of range.

File: blockchain/bridgeagg/state.go (hunk 1 above)
Comment (tin-incognito) [reply blockchain/bridgeagg/state.go]: Fixed

File: rpcserver/http_bridge_agg.go (hunk 3 above)
Comment (tin-incognito) [reply rpcserver/http_bridge_agg.go:41]: Automatically recover!

File: blockchain/bridgeagg/state.go (hunk 2 above)
Comment (tin-incognito) [reply blockchain/bridgeagg/state.go:105]: config can change birdgeagg state so should read it at the end of processing beacon block

Comment (0xkraken) [reply blockchain/bridgeagg/state.go:105]: I think should setup or update configs before processing other tasks (shielding, unshielding)
For example, there is the first converting tx created in the block that initialing unified token configs, in this case, the converting tx will be rejected.

File: rpcserver/http_bridge_agg.go (hunk 3 above)
Comment (0xkraken) [reply rpcserver/http_bridge_agg.go:41]: don't make nodes panic!

Comment (tin-incognito) [reply rpcserver/http_bridge_agg.go:41]: Fixed

File: blockchain/bridgeagg/state.go (hunk 2 above)
Comment (tin-incognito) [reply blockchain/bridgeagg/state.go:105]: Explained

**Pull request closed without merging**